### ISSUE TITLE
version 85.8 doesn't fix backgrounds

### ISSUE THREAD

**Issue by carlwgeorge** [CentOS/centos-logos#2]
In [rhbz#1974149](https://bugzilla.redhat.com/show_bug.cgi?id=1974149) we're attempting to clean up the background images in anaconda.  I wanted to move the conversation here temporarily for easier image uploads/previews.

Here is what centos-logos-85.5-1.el8 looked like in CentOS-Stream-8-x86_64-20210617-boot.iso.

![sidebar-before](https://user-images.githubusercontent.com/12187228/123866651-f93ad600-d8f2-11eb-838d-1efecd2e8c7d.png)

Here is what centos-logos-85.8-1.el8 looks like in CentOS-Stream-8-x86_64-20210628-boot.iso (a test compose ISO).

![sidebar-after](https://user-images.githubusercontent.com/12187228/123866658-fc35c680-d8f2-11eb-80b3-7a7697dfc046.png)

**Comment by carlwgeorge** [comment]
Here is a before and after from the same ISOs of the top bar.

![topbar-before](https://user-images.githubusercontent.com/12187228/123868936-fa213700-d8f5-11eb-9c59-1d08c5ae7e35.png)

![topbar-after](https://user-images.githubusercontent.com/12187228/123868942-fdb4be00-d8f5-11eb-8d5c-71bb08f6f7bc.png)

**Comment by areguera** [comment]
The background images were updated, so during the compose they were copied. Background images are ok.

The CSS file the centos-logos package provides to control the background image appearance didn't replace the one provided by anaconda package at `final.iso:/images/install.img:/usr/share/anaconda/anaconda-gtk.css`. This indicates that it was not installed during the compose, so the css file anaconda package provides took the place.

My guess is that by removing the action of copying anaconda-gtk.css file from anaconda package, and doing it from centos-logo package the `final.iso:/images/install.img:/usr/share/anaconda/anaconda-gtk.css` should have the expected information.

**Comment by carlwgeorge** [comment]
I think I found the problem.  The RHEL product data (which the CentOS Stream product data inherits from) was updated to pull in the custom_stylesheet [here](https://github.com/rhinstaller/anaconda/commit/6952955ae67b56c6fd00674def2f0ddde07e4ea6).  That change is part of version 34.24.4, but we're still on version 33.16.5.1.  We'll need to apply it as a downstream patch, as well as requesting upstream backport it to the rhel-8 branch.  Until that's complete, we'll need to put this on hold.

**Comment by carlwgeorge** [comment]
anaconda-33.16.5.2-1.el8 was recently pushed, so I used the opportunity to apply https://github.com/rhinstaller/anaconda/pull/3477 as a patch.  I generated a new compose with that build and centos-logos-85.8-1.el8, and it looks great now.

![sidebar-after2](https://user-images.githubusercontent.com/12187228/124325746-d1de4600-db4a-11eb-86d0-068e9f3f5b69.png)

![topbar-after2](https://user-images.githubusercontent.com/12187228/124325771-db67ae00-db4a-11eb-87c3-095a4af5e8ec.png)

Thanks for your work on this @areguera!  I'll either push out this compose, or do a new compose and push that early next week.  Either way, the fix will be in the public CS8 iso very soon.
